packages/ckeditor5-enter/src/enterobserver.js

```diff
@@ -23,7 +23,7 @@ export default class EnterObserver extends Observer {
 		const doc = this.document;
 
 		doc.on( 'keydown', ( evt, data ) => {
-			if ( this.isEnabled && data.keyCode == keyCodes.enter ) {
+			if ( this.isEnabled && data.keyCode == keyCodes.enter && !data.ctrlKey && !data.metaKey ) {
```

Comment:
It's because `editor.keystrokes` is not stopping the original event so it splits the list item while pressing `Cmd+Enter`. Now I'm wondering if we should change the `KeystrokeHandler` behavior or this code.

Reply:
I'm changing it, to-do list will use 'keydown' listener so it can be stopped in the context of to-do list.